modules/core/src/main/scala/org/scalasteward/core/buildtool/mill/MillAlg.scala

```diff
@@ -47,9 +48,14 @@ final class MillAlg[F[_]](implicit
         val command = Nel("mill", List("-i", "-p", predef.toString, "show", extractDeps))
         processAlg.execSandboxed(command, buildRootDir)
       }
-      parsed <- F.fromEither(
-        parser.parseModules(extracted.dropWhile(!_.startsWith("{")).mkString("\n"))
-      )
+      jsonString = extracted.dropWhile(!_.startsWith("{")).mkString("\n")
+      _ <- F.whenA(jsonString.isBlank) {
```

Comment:
`.isBlank` is added in JDK 11.
Since we still support JDK 8, it may be replaced with `jsonString.trim.isEmpty`.

Reply:
Also please add test cases.